enhancements/20250820-support-multipath.md

```diff
@@ -0,0 +1,290 @@
+# Support Multipath Disk
+
+Support for multipath disk devices in Harvester to enable reliable storage access through multiple paths for improved fault tolerance and performance.
+
+## Summary
+
+This enhancement introduces multipath device recognition and management in Harvester's node-disk-manager component, allowing users to configure and utilize multipath storage devices safely while ensuring proper identification and avoiding conflicts with existing Longhorn v2 and LVM DM-devices.
+
+### Related Issues
+
+- Primary Issue: https://github.com/harvester/harvester/issues/6975
+- Non-WWN Device Handling Issue: https://github.com/harvester/harvester/issues/6261 (out of scope for this enhancement)
+
+## Motivation
+
+### Goals
+
+- Enable Harvester to recognize and manage multipath disk devices properly
+- Support multipath device configuration in Harvester installer with extended configuration options
+- Distinguish between multipath devices and Longhorn v2 DM-devices to avoid conflicts
+- Ensure multipath devices are back after starting multipathd service when multipathd service is disabled.
```

Comment:
```suggestion
- Ensure multipath devices are back after starting multipathd service when multipathd service is enabled.
```

Reply:
Ensuring that multipath devices are available should be solely the responsibility of multipathd, so this may not need to be listed here as a goal?

Reply:
Yeah, good point. It's multipath service stuff. We can remove this.